### PR TITLE
[TECHNICAL-SUPPORT] LPS-152341 Requested Content-Type and received Response do not match

### PR DESCRIPTION
**Pull request by chileces**
In order to make the requested content type and received response match, i just change the way we write the response for the HTTP request by using PortletResponseUtil to send the file with content-type "image/png" instead of "text/html" as the default

### CODE REVIEW

**Comment by chileces** [comment]
@hongvo2308 please help me to review this ticket https://issues.liferay.com/browse/LPS-152341
thank you

File: modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/portlet/action/ApplicationsMenuLiferayLogoMVCResourceCommand.java [hunk 1]
```diff
@@ -68,8 +68,9 @@ protected void doServeResource(
 		if (inputStream == null) {
 			return;
 		}
-
-		PortletResponseUtil.write(resourceResponse, inputStream);
+		PortletResponseUtil.sendFile(
```
Comment (ces-hongvo) [inline modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/portlet/action/ApplicationsMenuLiferayLogoMVCResourceCommand.java]: Hi @chileces 
Could you please try the below solution 
```
resourceResponse.setContentType(ContentTypes.IMAGE_PNG);
PortletResponseUtil.write(resourceResponse, inputStream);
```
Thank you.

**Review by hongvo2308 — CHANGES_REQUESTED** [review]

File: modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/portlet/action/ApplicationsMenuLiferayLogoMVCResourceCommand.java (hunk 1 above)
Comment (chileces) [reply modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/portlet/action/ApplicationsMenuLiferayLogoMVCResourceCommand.java]: @hongvo2308 thank you for your suggestion, i did it

Comment (chileces) [reply modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/portlet/action/ApplicationsMenuLiferayLogoMVCResourceCommand.java]: @hongvo2308 code formatted

**Comment by ces-hongvo** [comment]
Forwarded https://github.com/liferay-ts-reviewer/liferay-portal/pull/49
Thank you.

**Pull request closed without merging**